obelisk_ws/src/obelisk_msgs/obelisk_sensor_msgs/msg/TrueSimState.msg

```diff
@@ -1,2 +1,20 @@
 string MESSAGE_NAME="TrueSimState"
 float64[] y
+
+std_msgs/Header header
+
+# Quantities required for viz
+float64[] q_base
+string base_link_name
+
+float64[] q_joints
+string[] joint_names
+
+# Velocities
+float64[] v_base
+float64[] v_joints
+
+# Contact forces
+# Vector3[] contact_forces
+
+# Applied control
```

Comment:
reminder that these things exist but are commented out - maybe remove them from this pr?